dev-docs/publisher-api-reference.md

```diff
@@ -396,6 +397,37 @@ Accepted values:
 | CPM <= $20 and >$8 | 	$0.50 increments             | $14.26 floored to $14.00 |
 | CPM >  $20 | 	Caps the price bucket at $20 | $24.82 floored to $20.00 |
 
+<a name="customCPMObject"></a>
+
+#### Custom CPM Bucket Sizing
+
+To set up your own custom CPM buckets, create an object like the following, and pass it into `setPriceGranularity`:
+
+```javascript
+const customConfigObject = {
+  "buckets" : [{
+      "precision" 2,  //default is 2 if omitted - means 2.1234 rounded to 2 decimal places = 2.12
+      "min" : 0,
+      "max" : 5,
+      "increment" : 0.01,
```

Comment:
Thanks for the catch, i write too much perl (where this is allowed).  Addressed this and the other trailing comma in https://github.com/prebid/prebid.github.io/commit/34dfc97f8668327a1497f4111ad41aa043305710